hummingbot/connector/exchange/bittrex/bittrex_in_flight_order.pyx

```diff
@@ -26,6 +26,9 @@ cdef class BittrexInFlightOrder(InFlightOrderBase):
             initial_state
         )
 
+        self.trade_id_set = set()
+        self.fee_asset = self.quote_asset
```

Comment:
I don't think it is right to assume that the fee asset is the quote asset. We should deduce this from the order side.

Reply:
The current estimated fee is always calculated as an amount in quote asset. I could not find any place in the documentation from Bittrex that suggest the fee is charged in the base asset. If we intend to change that we should have some validation from Bittrex, either in the form of documentation or by their confirmation.